src/bindings/python/src/openvino/runtime/utils/decorators.py

```diff
@@ -27,6 +28,31 @@ def wrapper(*args: Any, **kwargs: Any) -> Node:
     return wrapper
 
 
+def _apply_affix(node: Node, prefix: str = "", suffix: str = "") -> Node:
+    node.friendly_name = prefix + node.friendly_name + suffix
+    return node
```

Comment:
Do we expect cases where same inputs will be shared by multiple ops that will use prefix/suffix decorator? If so, maybe we should try to detect repetitions of prefix/suffix to not append/prepend them multiple times?

Reply:
I am resolving this as new state of implementation only appends to autogenerated nodes.